lib/manager/npm/post-update/yarn.ts

```diff
@@ -166,6 +178,10 @@ export async function generateLockFile(
       }
     });
 
+    commands = commands.map((c) =>
+      c.startsWith('yarn') ? `${c} ${cmdOptions}`.trim() : c
+    );
```

Comment:
I don't like this change, make code more complicate

Reply:
Reverted.